unit_tests/system/python_loader_test.cpp

```diff
@@ -49,23 +49,29 @@ TEST(PythonLoader, print_error) {
         "Message: global name 'a' is not defined\nTraceback");
 }
 
+
```

Comment:
Do you mean during Python copy? Packed Python packages folder has `21 MB` I don't see any big benefit by reducing size even more. Further discussion may be required on this topic.